observations/obs_converters/MODIS/shell_scripts/MOD29E1D_hdf_to_netcdf.py

```diff
@@ -0,0 +1,44 @@
+import rioxarray as rxr
+import xarray as xr
+import geopandas as gpd
+import numpy as np
+import sys 
+
+hdf_file = sys.argv[1]
+year = sys.argv[2]
+month = sys.argv[3]
+day = sys.argv[4]
+
+pre_file = rxr.open_rasterio(hdf_file)
+
+# translate to geopandas to extract lat/lon from the hdf file's EASE-Grid CRS
+mid_file = gpd.GeoDataFrame(data = pre_file.Ice_Surface_Temperature_NP[0], 
+                            geometry=gpd.points_from_xy(pre_file.x, pre_file.y),
+                            crs='EPSG:3408')
+
+laty = mid_file.to_crs('EPSG:4326').geometry.y
+lonx = mid_file.to_crs('EPSG:4326').geometry.x
+
+laty = laty.where((laty <= 90) & (laty >= 0))
+lonx = lonx.where((lonx <= 180) & (lonx >= -180))
+
+lon, lat = np.meshgrid(lonx, laty)
+
+pre_file['tsfc'] = pre_file.Ice_Surface_Temperature_NP[0]/100 - 273.15
+pre_file['lat'] = ({'x','y'}, lat)
+pre_file['lon'] = ({'x','y'}, lon)
```

Comment:
I think x,y is backwards here or in MOD29E1D_to_obs.f90  looking at the output netcdf file:

```
netcdf test {
dimensions:
        y = 1906 ;
        x = 1328 ;
variables:
        double tsfc(y, x) ;
                tsfc:_FillValue = NaN ;
        double y(y) ;
                y:_FillValue = NaN ;
        double lat(x, y) ;
                lat:_FillValue = NaN ;
        double lon(x, y) ;
                lon:_FillValue = NaN ;
        double x(x) ;
                x:_FillValue = NaN ;
```

https://github.com/NCAR/DART/blob/420be65cb451d2b87caab2a98b1f5e347ec64b98/observations/obs_converters/MODIS/MOD29E1D_to_obs.f90#L118-L121